.circleci/config.yml

```diff
@@ -12,6 +12,14 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - pipenv_install_36_browser:
```

Comment:
I think these can be omitted - Unless you are just trying to confirm that the deps can be installed with this version of docker?

Reply:
Ah ok. so I can remove `pipenv_install_36_browser` and `pip_install_36_browser` jobs.

I guess essentially they duplicate the `pipenv_install_36` and `pip_install_36` jobs anyway.